jest.e2e.config.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  verbose: true,
+  preset: 'jest-playwright-preset',
+  testMatch: [
+    '**/test/e2e/**/?(*.)+(test).+(ts|js)',
+  ],
+  transform: {
+    '^.+\\.ts$': 'ts-jest',
+  },
+  testEnvironmentOptions: {
+    'jest-playwright': {
+      browsers: ['chromium', 'firefox'],
```

Comment:
Will these be installed on CI/CD pipeline?

Reply:
hmm, that is a good question.
I think we will have to install it on the environment before running the pipeline.
I'll check.

Reply:
So, yea, `firefox` and `chromium` were already present on the ubuntu environment the pipeline runs on.
I found [this](https://github.com/microsoft/playwright-github-action), which allows playwright to run on `firefox`, `chromium` and `webkit`.
I've added it to the workflow and enabled `webkit` on the playwright configs target array, so now it runs on it too. =D